DEPS

```diff
@@ -0,0 +1,85 @@
+use_relative_paths = True
+
+vars = {
+  'chromium_git': 'https://chromium.googlesource.com',
+  'github': 'https://github.com',
+
+  'buildtools_revision': 'ab7b6a7b350dd15804c87c20ce78982811fdd76f',
+  'clang_revision': 'abe5e4f9dc0f1df848c7a0efa05256253e77a7b7',
+  'effcee_revision': '04b624799f5a9dbaf3fa1dbed2ba9dce2fc8dcf2',
+  'googletest_revision': '98a0d007d7092b72eea0e501bb9ad17908a1a036',
+  're2_revision': '6cf8ccd82dbaab2668e9b13596c68183c9ecd13f',
+  'spirv_headers_revision': '3ce3e49d73b8abbf2ffe33f829f941fb2a40f552',
```

Comment:
We'll need a defined process for updating these commit hashes.  I'm not saying it has to be now now now, but soon.